user_docs/en/userGuide.t2t

```diff
@@ -58,7 +60,7 @@ NVDA is bundled with [eSpeak NG https://github.com/espeak-ng/espeak-ng], a free,
 Information about other speech synthesizers that NVDA supports can be found in the [Supported Speech Synthesizers #SupportedSpeechSynths] section.
 
 ++ Braille support ++[BrailleSupport]
-For users that own a refreshable braille display, NVDA can output its information in braille.
+For users that own a refreshable braille display, NVDA uses the open source braille translator [Liblouis https://github.com/liblouis/liblouis] to output its information in braille.

```

Comment:
I think this is a little misleading, as we use many different technologies to output information in braille.
Also, this should link to the main LibLouis website

```suggestion
For users that own a refreshable braille display, NVDA can output its information in braille.
NVDA uses the open source braille translator [LibLouis https://liblouis.io/] to generate braille sequences from text.
```